modules/module02.md

```diff
@@ -15,6 +15,7 @@ For more information about linked services, see [Linked services in Azure Data F
 | --- | --- |
 | 1 | [Create ADLS Linked Service with Managed VNet IR](#1-create-adls-linked-service-with-managed-vnet-ir) |
 | 2 | [Create ADLS Linked Service with default Azure IR](#2-create-adls-linked-service-with-default-azure-ir) |
+| 3 | [Create Azure SQL DB Linked Service](#3-create-azuresqldb-linked-service) |
```

Comment:
Do you still need this linked service? I don't see a section 3 named this in the file? @HaoZhang615